client/lib/wp/sync-handler/whitelist-handler.js

```diff
@@ -8,7 +8,8 @@ const debug = debugFactory( 'calypso:sync-handler:whitelist' );
 const whitelist = [
 	/^\/me\/posts$/,
 	/^\/me\/settings/,
-	/^\/sites\/[\w.]+\/posts/
+	/^\/sites\/[\w.]+\/posts/,
+	/^\/videos/
```

Comment:
[Sync-handler](https://github.com/Automattic/wp-calypso/tree/master/client/lib/wp/sync-handler) is used for calypso offline. It intercepts client requests and stores server responses in local storage. It is work in progress (e.g. cache invalidation is not done, etc.) and I do not think that we have to whitelist the `videos` endpoint at this time. How offline video preview works (if it can work at all) can be evaluated and this endpoint whitelisted later. I removed it from the whitelist for the time being. (Earlier I added it because the "not whitelisted: skip "/videos/" request" implied to me that my request was blocked, that, however, was a wrong interpretation on my end.)